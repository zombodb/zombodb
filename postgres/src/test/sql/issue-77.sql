SELECT term
FROM zdb_suggest_terms('so_posts', 'body', 'beer', '', 5000)
LIMIT 9;