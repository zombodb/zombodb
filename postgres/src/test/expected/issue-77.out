SELECT term
FROM zdb_suggest_terms('so_posts', 'body', 'beer', '', 5000)
LIMIT 9;
 term  
-------
 beer
 been
 bear
 beef
 beers
 beep
 bees
 beter
 beed
(9 rows)

